65.md

```diff
@@ -6,59 +6,136 @@ Relay List Metadata
 
 `draft` `optional`
 
-Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
+Defines two replaceable events to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+Relay Usages
+------------
+
+There are three defined usages for a relay:
+
+- `INBOX` relays are for events that the relay list owner will be reading, specifically watching for events that tag them. These do not include DMs (except in the deprecated form of relay list).
+
+- `OUTBOX` relays are for events that the relay list owner has authored and is publishing for public consumption.
+
+- `PRIVATE INBOX` relays are like INBOX relays, except they are for reciept of private messages such as perhaps direct messages (DMs), giftwraps, or the like (but this NIP does not specify what kinds should use relays with this designator).
+
+### When to use which
+
+When posting events (except private ones):
+
+- Clients SHOULD post events to all of their client user's OUTBOX relays,
+- For each p-tag in a posted event, clients SHOULD post the event to at least some of the INBOX relays of the tagged person.
+
+When posting private messages:
+
+- Clients SHOULD post events to at least some (preferably all) of the the p-tag recipient's PRIVATE INBOX relays.
+- Clients SHOULD post a copy to their client user's own PRIVATE INBOX relays.
+- If a user has no PRIVATE INBOX relays, INBOX relays should be used instead.
+
+When seeking events from a user:
+
+- Clients SHOULD seek them from at least some of that user's OUTBOX relays.
+
+When seeking events about a client's user (where the user was tagged):
+
+- Clients SHOULD seek them from all of their user's INBOX relays.
+
+When seeking private messages for a client's user:
```

Comment:
Guidance here should emphasize that Clients should only connect to Private Inbox if they can AUTH the user they are requesting information for.